ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/services/DefaultKsqlClient.java

```diff
@@ -137,4 +129,9 @@ private KsqlTarget getTarget(final KsqlTarget target, final Optional<String> aut
         .map(target::authorizationHeader)
         .orElse(target);
   }
+
+  private static HttpClientOptions createClientOptions() {
+    return new HttpClientOptions().setMaxPoolSize(100);
```

Comment:
I'm not familiar with HTTP connection pooling -- is 100 a typical pool size? Feels large.

Reply:
Maybe, but it's just an upper limit, connections won't be created unless they're required.